source/MissionTestMod/TestMod.cs

```diff
@@ -72,7 +75,16 @@ protected override void OnSubModuleLoad()
               "Join Online Tavern",
               new TextObject("Join Online Tavern"),
               9991,
-              StartClientInTavern,
+              () =>
```

Comment:
Move this into it's own function and this is under "Join Tavern". If you could make it work for both